examples/aerospike-cluster-values.yaml

```diff
@@ -0,0 +1,103 @@
+## Aerospike cluster size
+replicas: 3
+
+## Aerospike Database Docker image
+image:
+  repository: aerospike/aerospike-server-enterprise
+  tag: 7.1.0.0
+
+## Aerospike access control configuration
+aerospikeAccessControl:
+  users:
+    - name: admin
+      secretName: auth-secret
+      roles:
+        - sys-admin
+        - user-admin
+
+## Aerospike Configuration
+aerospikeConfig:
+  service:
+    feature-key-file: /etc/aerospike/secret/features.conf
+  security: {}
+
+  network:
+    service:
+      port: 3000
+    heartbeat:
+      port: 3002
+    fabric:
+      port: 3001
+
+  namespaces:
+    - name: test
+      replication-factor: 2
+      storage-engine:
+        type: device
+        devices:
+          - /test/dev/xvdf
+    - name: testMem
+      replication-factor: 1
+      storage-engine:
+        type: memory
+        data-size: 1073741824
+
+## Network policy
+aerospikeNetworkPolicy: {}
+
+## Pod spec
+podSpec:
+   multiPodPerHost: true
+
+## Rack configuration
+rackConfig:
+  namespaces:
+    - test
+
+## Storage configuration
+storage:
+  filesystemVolumePolicy:
+    cascadeDelete: true
+    initMethod: deleteFiles
+  blockVolumePolicy:
+    cascadeDelete: true
```

Comment:
Since attached block storage might not be clean on first boot, initMethod: dd must clean it up.

More on this: https://aerospike.com/docs/cloud/kubernetes/operator/configuration/Cluster-configuration-settings#blockfilesystem-volume-policy

Reply:
done, please review